styles/style.css

```diff
@@ -620,3 +621,193 @@
   margin-top: 15px;
 }
 /* Cta Section End  */
+
+/* Cta Section Responsiveness Start */
+
+@media screen and (max-width: 1500px) {
+  .cta-section{
```

Comment:
telefondaki margin ve padding değerlerini sildikten sonra  padding-block: 50px; gibi bir değer vererel cta section için alttan ve üstten boşluk ekleyebilirsin